### PR TITLE
Added Readme for examples and updated QSG

### PR DESCRIPTION
**Pull request by csteavu**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-instrument-addon-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Added Readme for examples and updated QSG with the updated snips and description for them

### Why should this Pull Request be merged?

No Readme and outdated example snips

### What testing has been done?

N/A

### CODE REVIEW

File: Source/Examples/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# **The Instrument Addon Scripting API Example project contains 5 VIs (one basic example, two aditional VIs that build on it and two protocol related examples) that showcase the API usage:**
+
+## 1) A basic example of configuring an Instrument Addon with a single message, with both Command and Response, and then exporting the configuration to a file on disk.
+   - It creates a new TCP Protocol object which is used as an input for a new Instrument Addon Custom Device (the Instrument Addon name is also changed)  
+   - Then a message section and a new message are created
+   - Command and Response sections are added, then configured
+   - the "settings" input for the "Configure Response" uses a a known value (42 in this case) as the channel refnum for the linked channel(composed of a variable name and a channel refnum) 
+   
+## 2) Example 1 covers a scenario where the user wants to alter the IP Address of a protocol and leave other properties unchanged
+   - The VI fetches the protocol of a Instrument Addon Custom Device (based on a refnum used as input), then it decides if the Protocol is TCP or AK TCP
```
Comment (phutanu) [inline Source/Examples/README.md:10]: "protocol of a**n** Instrument Addon"
"then it detects if the Protocol..."

File: Source/Examples/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## 2) Example 1 covers a scenario where the user wants to alter the IP Address of a protocol and leave other properties unchanged
+   - The VI fetches the protocol of a Instrument Addon Custom Device (based on a refnum used as input), then it decides if the Protocol is TCP or AK TCP 
+   - It then alters the IP Address and inputs the value to the appropriate protocol factory
+   - The mentioned factory produces a protocol object that is used as an input for the Set Protocol node  
+   - Thus, the Instrument Addon is now updated with the desired IP Address
+ 
+## 3) Example 2 just sets a new Protocol, regardless of the previous configuration
+
+## 4) Example 3 builds on the Basic Example as follows:
+   - It uses a configuration file (exported in the Basic Example) to create a Instrument Addon Custom Device with the same configuration
```
Comment (phutanu) [inline Source/Examples/README.md:18]: "to create a**n** Instrument Addon Custom Device"

File: Source/Examples/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+   - It then adds a Global MetaData Channel and another Message
+   - For the newly added message, it adds a Command Section and a MetaData Channel  
+   - It configures the Command to use the MetaData Channel and the Global MetaData Channel by assigning their refnums to two variable names (thus creating two linked channels )
+   - Finally, the new configuration is exported to the disk
+   
+## 5) Example 4 builds on Example 3 as follows:
+   - It uses a configuration file (exported in Example 3) to create a Instrument Addon Custom Device with the same configuration
```
Comment (phutanu) [inline Source/Examples/README.md:25]: "to create a**n** Instrument Addon Custom Device"

File: Source/Examples/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+   - It then adds a Response section to the message added in example 3 
+   - A Data Channel is also added
+   - Finally, the Response is configured to use the Data Channel Refnum
+   
+# **Instructions on how to run the example:**
```
Comment (phutanu) [inline Source/Examples/README.md:30]: "the example**s**

File: Source/Examples/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+### 1) Open the Instrument Addon Scripting API Examples.lvproj
+### 2) Open the desired VI and modify the message type, settings, config file paths as desired and run the VI 
+#### **Note:** It is strongly advised to operate the modifications mentioned in step 2, as the logic in the VIs is for demonstration purposes only
+#### For example, you may need another wait type for your Response,other types of Channels, padding may not make sense in your usecase, etc.)
```
Comment (phutanu) [inline Source/Examples/README.md:35]: "Response,other" - a space is required after the comma, here.
- Is the end parenthesis intended?
- Maybe this last sentence may look better as a continuation of the Note.

File: Source/Quick Start Documentation/Instrument Addon Quick Start Guide.md [hunk 6]
```diff
@@ -256,29 +256,28 @@ This is a basic example of configuring an Instrument Addon with a single message
...
+
 ### Example 2:<br/>
 Set a new Protocol, regardless of the previous configuration<br/>
 
 ![Example2](./Images/image054.png)
 
 ### Example 3:<br/>
-This example uses a prevous configuration (from the first basic example), creates the Instrument Addon based on it,<br/>
- adds another message with both Command and Response, and then exporting the configuration to a file on disk.  <br/>
+This example uses a configuration file (exported in the Basic Example) to create a Instrument Addon Custom Device with the same configuration,
```
Comment (phutanu) [inline Source/Quick Start Documentation/Instrument Addon Quick Start Guide.md:272]: "to create a**n** Instrument Addon"

**Review by phutanu — APPROVED** [review]
Except the small things mentioned above that I would like to see addressed, this looks ok.

**Review by bogdanp-ni — APPROVED** [review]

**Pull request merged**